site/e2e/tests/manageUsers.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { test, expect } from "@playwright/test";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
+
+test("manage users", async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/users`, { waitUntil: "domcontentloaded" });
+  await expect(page).toHaveTitle("Users - Coder");
+
+  await page.getByRole("button", { name: "Create user" }).click();
+  await expect(page).toHaveTitle("Create User - Coder");
+
+  const userValues = {
+    username: "testuser",
+    email: "testuser@coder.com",
+    loginType: "password",
+    password: "s3cure&password!",
+  };
+  await page.getByLabel("Username").fill(userValues.username);
+  await page.getByLabel("Email").fill(userValues.email);
+  await page.getByLabel("Login Type").click();
+  await page.getByRole("option", { name: "Password", exact: false }).click();
+  // Using input[name=password] due to the select element utilizing 'password'
+  // as the label for the currently active option.
+  const passwordField = page.locator("input[name=password]");
+  await passwordField.fill(userValues.password);
+  await page.getByRole("button", { name: "Create user" }).click();
+  await expect(page.getByText("Successfully created user.")).toBeVisible();
+
+  const userRow = page.locator("tr", { hasText: userValues.email });
```

Comment:
"Delete user" should also be a separate function/workflow that starts with "click on Users tab"

Reply:
Ok, I will split the flows. When I usually write e2e tests I don't try to make them more atomic since it has a cost to recreate things.

Reply:
That is true, but on the other hand you can reuse it as `deleteUser("johndoe")`.